serializability-checker-compiler-plugin/src/test/resources/GenericsTest3.scala

```diff
@@ -0,0 +1,18 @@
+package org.random.project
+
+import akka.actor.typed.ActorSystem
+import akka.grpc.scaladsl.WebHandler
+import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
```

Comment:
Maybe you just need akka-http and not akka-http-cors on the test classpath?

Reply:
`grpcWebHandler` is defined like this:
```
def grpcWebHandler(handlers: PartialFunction[HttpRequest, Future[HttpResponse]]*)(
      implicit as: ClassicActorSystemProvider,
      corsSettings: CorsSettings = defaultCorsSettings): HttpRequest => Future[HttpResponse] = {
    implicit val system = as.classicSystem
    val servicesHandler = ServiceHandler.concat(handlers: _*)
    Route.toFunction(cors(corsSettings) {
      handleWith(servicesHandler)
    })
  }
  ```
  
  with 
  `import ch.megard.akka.http.cors.scaladsl.settings.CorsSettings`
  
  I need to add akka-http-cors to the classpath, otherwise an error occurs:
  ```
  error: Symbol 'type ch.megard.akka.http.cors.scaladsl.settings.CorsSettings' is missing from the classpath.
  ```

Reply:
Awww okay I see